File: Socket/src/main/java/com/cunoc/Map/Square.java
package com.cunoc.Map;

public class Square {

    private final int possX;
    private final int possY;
    private ListSquare type;

    public Square (ListSquare type,int possX,int possY){
        this.type = type;
        this.possX = possX;
        this.possY = possY;
    }

    public ListSquare getType() {
        return type;
    }

    
    public int getPossX() {
        return possX;
    }

    public int getPossY() {
        return possY;
    }

    public void setType(ListSquare type){
        this.type = type;
    }
    @Override
    public String toString() {
        return "PossX: "+this.possX+" PossY: "+this.possY + " Tipo : " +this.type.toString();
    }
    
}

File: Android/app/src/main/java/com/example/boxworld_sokoban/juego/JflexYCup/xml/Map/Square.kt
package com.example.boxworld_sokoban.juego.JflexYCup.xml.Map

import java.io.Serializable

class Square(type: ListSquare, possX: Int, possY: Int): Serializable {

    private var possX = 0
    private var possY = 0
    private var type: ListSquare = ListSquare.UNDEFINED

    init{
        this.type = type
        this.possX = possX
        this.possY = possY
    }

    fun getType(): ListSquare {
        return type
    }


    fun getPossX(): Int {
        return possX
    }

    fun getPossY(): Int {
        return possY
    }

    fun setType(type: ListSquare) {
        this.type = type
    }

    override fun toString(): String {
        return "PossX: " + possX + " PossY: " + possY + " Tipo : " + type.toString()
    }

}